jetson/nav/gate_search/gateStateMachine.cpp

```diff
@@ -225,53 +232,52 @@ NavState GateStateMachine::executeGateFace()
     return NavState::GateFace;
 } // executeGateFace()
 
-// Turn to furthest post, if possible
+// Turn to furthest post (or the only post if only one is available)
 NavState GateStateMachine::executeGateTurnToFarPost()
 {
-    if( mPhoebe->roverStatus().target2().distance > 0 ) 
+    if( mPhoebe->roverStatus().rightTarget().distance > 0 ) 
     {
-        if( mPhoebe->roverStatus().target().distance < mPhoebe->roverStatus().target2().distance ) 
+        if( mPhoebe->roverStatus().leftTarget().distance < mPhoebe->roverStatus().rightTarget().distance ) 
         {
-            if( mPhoebe->turn(mPhoebe->roverStatus().target2().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
+            if( mPhoebe->turn(mPhoebe->roverStatus().rightTarget().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
             {
                 return NavState::GateDriveToFarPost;
             }
         }
         else 
         {
-            if( mPhoebe->turn(mPhoebe->roverStatus().target().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
+            if( mPhoebe->turn(mPhoebe->roverStatus().leftTarget().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
             {
                 return NavState::GateDriveToFarPost;
             }   
         }
     }
     else 
     {
-        if( mPhoebe->turn(mPhoebe->roverStatus().target().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
+        if( mPhoebe->turn(mPhoebe->roverStatus().leftTarget().bearing + mPhoebe->roverStatus().odometry().bearing_deg) ) 
         {
             return NavState::GateDriveToFarPost;
         }
     }
-
     return NavState::GateTurnToFarPost;
 } // executeGateTurnToFarPost()
 
-// Drive to the farthest post, if possible
+// Drive to furthest post (or the only post if only one is available)
```

Comment:
If we only see one post, isn't this the wrong state to be in? In that case, we would rather be looking for a second post, right?

Reply:
I considered if there was only one post that we can see, that would be the "far" post we drive to (the code inside the executeGateDriveToFarPost function will determine if we only drive to a single post we see, or if we drive to a second one if it happens to be farther away). I can rename this if this is unclear

Reply:
I think it's fine to leave the name as is. Make sure you update any documentation now that we have new states.